### PR TITLE
Alarms for Navigator

### PR DESCRIPTION
**Pull request by mattlangford**
Contains some changes made during the sub trip (including command line missions).

### CODE REVIEW

File: simulation/sub8_gazebo/nodes/pedometry.txt [hunk 1]
```diff
@@ -1 +1 @@
-34989.8186173
\ No newline at end of file
+35424.0068616
```
Comment (DSsoto) [inline simulation/sub8_gazebo/nodes/pedometry.txt:1]: did you mean to commit this?

Comment (rleyva) [inline simulation/sub8_gazebo/nodes/pedometry.txt:1]: Remove this file from the git. Tracking a file that will be different for everyone, that's used to track how much its moved in sim is useless. Also, it makes it a pain to commit while the sim is running.

**Comment by DSsoto** [comment]
Looks fine. In the future, try to use less 'fuck' print statements. Merging.

File: simulation/sub8_gazebo/nodes/pedometry.txt (hunk 1 above)
Comment (mattlangford) [inline simulation/sub8_gazebo/nodes/pedometry.txt:1]: It was added to the gitignore and is not in the navigator repo

**Pull request merged**